src/ToolbarService.php

```diff
@@ -254,9 +255,19 @@ public function saveData(ServerRequest $request, ResponseInterface $response)
             'requested_at' => $request->getEnv('REQUEST_TIME'),
             'panels' => [],
         ];
-        /** @var \DebugKit\Model\Table\RequestsTable $requests */
-        $requests = $this->getTableLocator()->get('DebugKit.Requests');
-        $requests->gc();
+        try {
+            /** @var \DebugKit\Model\Table\RequestsTable $requests */
+            $requests = $this->getTableLocator()->get('DebugKit.Requests');
+            $requests->gc();
+        } catch (MissingDatasourceConfigException $e) {
+            Log::warning(
+                'Unable to save request. Check your debug_kit datasource connection ' .
```

Comment:
We should also add a big fat note in the readme asking people to check the error logs for details if Debugkit isn't showing up :).

Reply:
Done. We can always add more reasons in the future.